src/index.js

```diff
@@ -138,10 +138,15 @@ class Dropzone extends React.Component {
     this.isFileDialogActive = false;
 
     fileList.forEach((file) => {
-      if (this.fileAccepted(file) && this.fileMatchSize(file)) {
-        if (!disablePreview) {
+      if (!disablePreview) {
+        try {
           file.preview = window.URL.createObjectURL(file); // eslint-disable-line no-param-reassign
+        } catch (err) {
+          // Continue regardless of error
```

Comment:
I think `console.error` or `console.info` during development would be more appropriate then throwing an error.

With that said, we could provide a `onPreviewFailure` callback so the parent component could handle it however they saw fit. For example, they might want to display an error message to the user.